event_management/src/main/java/com/example/event_management/model/Events.java

```diff
@@ -0,0 +1,46 @@
+package com.example.event_management.model;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import lombok.*;
+
+@Entity
+@Table(name = "events")
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+public class Events {
+
+    @Id
+    @Column(name = "id")
+    private String id;
+
+    @Column(name = "name")
+    @NotBlank(message = "Add Event name!")
+    private String name;
+
+    @Column(name = "date")
+    @NotBlank(message = "Add Date!")
+    private String date;
+
+    @Column(name = "time")
+    @NotBlank(message = "Add Time!")
+    private String time;
+
+    @Column(name = "location")
+    @NotBlank(message = "Add Location!")
+    private String location;
+
+    @Column(name = "total_seats")
+    @NotBlank(message = "Add Number of Seats!")
+    private int totalSeats;
```

Comment:
not required as it is a primitive integer. By default, it will going to have 0 value.
```suggestion
    private int totalSeats;
```

Reply:
updated